src/IndexManager.php

```diff
@@ -239,6 +255,60 @@ private function setIndexIfMapping()
         $this->indexIfMapping = $mapping;
     }
 
+    /**
+     * For each chunk performs the provided operation.
+     *
+     * @param  \Doctrine\Common\Persistence\ObjectManager $objectManager
+     * @param  array $entities
+     * @param  callable $operation
+     * @return array
+     */
+    private function forEachChunk(ObjectManager $objectManager, array $entities, $operation)
+    {
+        $batch = [];
+        foreach (array_chunk($entities, $this->configuration['batchSize']) as $chunk) {
+            $searchableEntitiesChunk = [];
+            foreach ($chunk as $entity) {
+                $entityClassName = ClassUtils::getClass($entity);
+
+                $searchableEntitiesChunk[] = new SearchableEntity(
+                    $this->getFullIndexName($entityClassName),
+                    $entity,
+                    $objectManager->getClassMetadata($entityClassName),
+                    $this->normalizer,
+                    ['useSerializerGroup' => $this->canUseSerializerGroup($entityClassName)]
+                );
+            }
+
+            $batch[] = $operation($searchableEntitiesChunk);
+        }
+
+        return $this->formatBatchResponse($batch);
+    }
+
+    /**
+     * Returns the aggregators instances of the provided entities.
+     *
+     * @param  Doctrine\Common\Persistence\ObjectManager $objectManager
```

Comment:
Same as above, except that without a leading backslash, this becomes a class relative to the current namespace. I recommend shortening it.